app/addons/verifyinstall/stores.js

```diff
@@ -0,0 +1,101 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  'api',
+  'addons/verifyinstall/constants',
+  'addons/verifyinstall/actiontypes'
+],
+
+function (FauxtonAPI, Constants, ActionTypes) {
+
+  var VerifyInstallStore = FauxtonAPI.Store.extend({
+    initialize: function () {
+      this.reset();
+    },
+
+    reset: function () {
+      this._isVerifying = false;
+
+      // reset all the tests
+      this._tests = {};
+      var keys = ['CREATE_DATABASE', 'CREATE_DOCUMENT', 'UPDATE_DOCUMENT', 'DELETE_DOCUMENT', 'CREATE_VIEW', 'REPLICATION'];
+      _.each(keys, function (key) {
+        this._tests[Constants.TESTS[key]] = { complete: false };
+      }, this);
+    },
```

Comment:
something like 

```
this._tests = Object.keys(Constants.TESTS).reduce(function (acc, el) {
  acc[Constants.TESTS[key]] = { complete: false };
  return acc;
}, {});
```

should do the trick so you don't have to hardcode all the keys again here